Rules/CommonScripts/KillMessages.as

```diff
@@ -310,6 +333,47 @@ void onTick(CRules@ this)
 	{
 		feed.Update();
 	}
+
+	// display message in chat whenever a killstreak happens
+	for(int a = 0; a < getPlayerCount(); a++) 
+	{ 
+		CPlayer@ player = getPlayer(a);
+		CBlob@ blob = player.getBlob();
+
+		if (blob !is null && player !is null && blob.exists("killstreak"))
+		{
+			if(getGameTime() - blob.get_u32("kill time") > 180 && blob.get_u8("killstreak") > 4)
+			{
+				string multiKill;
```

Comment:
this chat stuff should be moved in a `isClient()` branch. it won't behave any differently really, but it makes the intent more clear (and avoids some minor unnecessary processing on the server side)